.github/workflows/pytest.yaml

```diff
@@ -8,20 +8,27 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
add matrix with windows, linux and macosx

Reply:
done